docs/src/whatsnew/latest.rst

```diff
@@ -164,7 +165,8 @@ This document explains the changes made to Iris for this release
 🔥 Deprecations
 ===============
 
-#. N/A
+#. `@bjlittle`_ dropped support for ``Python 3.7``, as per the `NEP-29`_
+   deprecation policy schedule. (:pull:`4481`)
```

Comment:
Deprecation or dependency? 🤔

Reply:
Good call, I've moved to under the `Dependencies` section instead :+1: